### ISSUE TITLE
SinonJS with mocha-webpack

### ISSUE THREAD

**Issue by mr-romeijn** [zinserjan/mocha-webpack#280]
Hello,

Is it possible to use sinonJS with this dependency? When using the spy method i get the following transpiling error: 

`ReferenceError: self is not defined`

I inspected the source and it seems that it uses it's own webpack conf to transpile the tests. However it does not seem to work with Sinon. I tried sinon version 5.0.1 and 6.0.1

**Comment by serut** [comment]
Yes we use it without any trouble. Our code is not fresh but is here : https://github.com/RegardsOss/regards-frontend/
